cmd/emp/dynos.go

```diff
@@ -58,6 +58,9 @@ func listDynos(w io.Writer) {
 func listDyno(w io.Writer, d *heroku.Dyno) {
 	listRec(w,
 		d.Name,
+		d.Id,
+		d.ContainerInstanceID,
+		d.EC2InstanceID,
```

Comment:
In the past, we've just altered the Go types directly, which I think is ok. We'll _eventially_ move away from re-using the Heroku API, but adding small things like this is fine.